src/core/processing/qgsnativealgorithms.cpp

```diff
@@ -80,10 +80,71 @@ void QgsNativeAlgorithms::loadAlgorithms()
   addAlgorithm( new QgsExtractByLocationAlgorithm() );
   addAlgorithm( new QgsFixGeometriesAlgorithm() );
   addAlgorithm( new QgsMergeLinesAlgorithm() );
+  addAlgorithm( new QgsSaveSelectedFeatures() );
   addAlgorithm( new QgsSmoothAlgorithm() );
   addAlgorithm( new QgsSimplifyAlgorithm() );
 }
 
+void QgsSaveSelectedFeatures::initAlgorithm( const QVariantMap & )
+{
+  addParameter( new QgsProcessingParameterVectorLayer( QStringLiteral( "INPUT" ), QObject::tr( "Input layer" ) ) );
+  addParameter( new QgsProcessingParameterFeatureSink( QStringLiteral( "OUTPUT" ), QObject::tr( "Selected features" ), QgsProcessing::TypeVectorPoint ) );
+}
+
+QString QgsSaveSelectedFeatures::shortHelpString() const
+{
+  return QObject::tr( "This algorithm creates a new layer with all the selected features in a given vector layer.\n\n"
+                      "If the selected layer has no selected features, all features will be added to the resulting feature." );
```

Comment:
* If the ~~selected~~ _input(?)_ layer (just to avoid too much selection in the sentence)
* added to the resulting ~~feature~~ layer